README.md

```diff
@@ -43,7 +43,9 @@ Get started with Rasa Pro in the browser using GitHub Codespaces.
      GitHub will show a notification, click on the green button to view the inspector where you can chat with your assistant.
 
 6. **Run Custom Actions:**
-   If your bot uses custom actions, you will need to run an action server. To do this:
-   - Create a second terminal window in the codespace
-   - load the env vars and activate your virtual environment: `source .env && source .venv/bin/activate`
-   - start the action server by running `rasa run actions`
+   If your bot uses custom actions, you can directly execute them on the Rasa Pro server. To do this, update your `endpoints.yml` file::
```

Comment:
this shouldn't be necessary, right? bc if they've followed the instructions to this point and run `rasa init --template tutorial`, this is already what their endpoints.yml will look like.

Reply:
I see, so this repo can't be used with any other assistant project other than the tutorial? I would amend to say `double-check your endpoints contains...`.

Reply:
yep I think that approach makes sense.